.env.example

```diff
@@ -41,3 +41,4 @@ ROLLBAR_TOKEN=
 MONGO_DB_URI=
 WEB3_SITE_URL=
 DISABLE_FACE_VERIFICATION=
+ENABLE_MONGO_LOCK=false
```

Comment:
@StanislavShevchenko no need to put values for variables in .env.example file

Reply:
[done](https://github.com/GoodDollar/GoodServer/pull/68/commits/1d02ca129b9addb3145466fe576deffbb5068ac0)